Server/src/migrations/SeedDatabase.ts

```diff
@@ -309,7 +309,10 @@ export class SeedDatabase1611344612000 implements MigrationInterface {
 
     await queryRunner.query('INSERT INTO accounts_datasets_dataset (accountsId, datasetId) VALUES (1, 2)');
 
-    let newGraph = new Savedgraphs();
+    // Temporary line, will be removed on merge
+    await queryRunner.query('ALTER TABLE savedgraphs RENAME TO graphstate');

```

Comment:
I think we can manually run alter tables

Reply:
We can, which is what I will do right before merging. I have to include these lines in the migration, as otherwise I would have to edit the test database for _everyone_ which would cause travis tests to fail for all other branches until this is merged.